pkg/hintrunner/utils/math_utils.go

```diff
@@ -26,6 +26,23 @@ func EcDoubleSlope(pointX, pointY, alpha, prime *big.Int) (big.Int, error) {
 	return Divmod(n, m, prime)
 }
 
+func LineSlope(point_aX, point_aY, point_bX, point_bY, prime *big.Int) (big.Int, error) {
+	// belongs to the line_slope method
```

Comment:
could you provide a link for this functionality as in AsInt example?

Reply:
Hi @MaksymMalicki , here it is https://github.com/lambdaclass/cairo-vm_in_go/blob/31c3628bc10ebc1628685b3cdfa72d0e938b533e/pkg/builtins/ec_op.go#L258

Reply:
I mean can you paste it in the code, so that the reader knows where does this functionality come from. Thanks!

Reply:
Okay